src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -428,6 +428,17 @@ default String dynamicConfigUrl() {
         return "";
     }
 
+    @ConfigItem(
+        keyName = "includeClientFrame",
+        name = "Include Client Frame",
+        description = "Whether to include the client frame within screenshots.",
```

Comment:
nit: I think this reads better
```suggestion
        description = "Whether to include the client frame in screenshots.",
```